### PR TITLE
AWS Backend Deploy

### PR DESCRIPTION
**Pull request by justinkonecny**

### CODE REVIEW

File: docs/aws/backend-deployment.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+   - Set the `Master username` to `postgres` and enter a unique password
+   - Adjust `Maximum storage threshold` to a reasonable value (e.g. `100` GiB)
+   - Under `Connectivity`, set `Virtual private cloud (VPC)` to `VPC-Public`
+   - For `Subnet group`, select `db-sbg-public`
+   - For `Public access`, select `Yes`
+   - For `Existing VPC security groups` select `SG-RDS`
+   - Under `Database authentication options`, select `Password authentication`
+   - Adjust any other default settings on this page, as necessary
+   - Click `Create database`
+ - Connect to the RDS instance, and create a new database for the project
```
Comment (jackblanc) [inline docs/aws/backend-deployment.md:24]: Do we have to manually run a `create database 'name';` every time? What if we just added that to the first migration 🤔

File: docs/aws/backend-deployment.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Elastic Beanstalk Setup
+ - Navigate to `Elastic Beanstalk` from the `Services` dropdown
+ - Click `Applications` from the left sidebar
```
Comment (jackblanc) [inline docs/aws/backend-deployment.md:28]: Do we not want a new application for each?

**Review by jackblanc — APPROVED** [review]
Thanks 🔥

File: docs/aws/backend-deployment.md (hunk 1 above)
Comment (justinkonecny) [reply docs/aws/backend-deployment.md:24]: I'm not sure--instinct says we *can't* do that but I'm not entirely sure?

File: docs/aws/backend-deployment.md (hunk 2 above)
Comment (justinkonecny) [reply docs/aws/backend-deployment.md:28]: I don't see any reason to have a new app for each

**Pull request merged**